docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
```

Comment:
strings like "10oct2016" or "3pm tdy 6pm" needs lots of work to be done in the parser.
since I am still working on the fundamentals functionality of the parser, currently I do not have any idea on how we could parse strings mentioned above